rudder-agent/debian/control

```diff
@@ -8,7 +8,7 @@ Homepage: http://www.rudder-project.org
 
 Package: rudder-agent
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, uuid-runtime, dmidecode, cron, net-tools, openssl
+Depends: ${shlibs:Depends}, ${misc:Depends}, uuid-runtime, dmidecode, cron, net-tools, openssl ${rudderDepends}
```

Comment:
It's a mistake, I tried to make this file dynamically built, but it need a lots of variables to work so I abandoned the idea